src/app/app.js

```diff
@@ -250,16 +272,10 @@ class App extends Component {
         </div>
 
         <div className="body">
-          {hash === 'display' ?
-            <DataDisplay projectMap={projectMap} /> :
-            hash === 'entry' ?
-              <DataEntry description={description} name={name} /> :
-                hash === 'assign-products' ?
-                  <NotImplemented name="AssignProducts" /> :
-                    hash === 'add-observations' ? <AddObservations /> : null}
+          {router()}
         </div>
 
-        <hr />
+        <hr style={{marginTop: '500px'}} />
```

Comment:
This styling should go in `app.scss`. I'll merge this and you can change it in your new branch off of `launchpad-ui`.